weaviate/batch/crud_batch.py

```diff
@@ -602,12 +608,19 @@ def _create_data(
         weaviate.UnexpectedStatusCodeException
             If weaviate reports a none OK status.
         """
+
+        params = None
+        if self._consistency_level is not None:
+            params = {"consistency_level": self._consistency_level}
```

Comment:
just as a hint, you can do
`param = None if self._consistency_level is None else {"consistency_level": self._consistency_level}`

no need to change it now

Reply:
here is a shorter one `self._consistency_level = ConsistencyLevel(x).value if x else None`